src/panels/lovelace/cards/hui-map-card.ts

```diff
@@ -453,6 +569,29 @@ class HuiMapCard extends LitElement implements LovelaceCard {
     }
   }
 
+  private async _getHistory(): Promise<void> {
+    const entityIds = this._configEntities
+      ?.map((entity) => entity.entity)
```

Comment:
This variable is always there ? If it's not , we shouldn't request the history of all entities over a period of time. In that case we shouldn't fetch anything.